pkg/csi_driver/gcfs_driver.go

```diff
@@ -132,6 +132,7 @@ type FeatureStateful struct {
 
 type FeatureLockRelease struct {
 	Enabled bool
```

Comment:
also if we adding variable for v2 we can as well change this variable `Enabled` to make it consistent

Reply:
discussed in comment above